system-addon/lib/ActivityStream.jsm

```diff
@@ -58,9 +114,11 @@ this.ActivityStream = class ActivityStream {
     this.options = options;
     this.store = new Store();
     this.feeds = feeds;
+    this._defaultPrefs = new DefaultPrefs(DEFAULT_PREFS);
```

Comment:
This reads kinda odd (assign default prefs with a new default prefs using default prefs), but I suppose they are indeed distinguishable, so we can just leave it. ;)